model/titleFilter.php

```diff
@@ -0,0 +1,18 @@
+<?php
+require_once "filterDecorator.php";
+
+class TitleFilter extends FilterDecorator
+{
+    public $title;
+    private $Filter;
+    public function __construct($Filter)
+    {
+        $this->Filter = $Filter;
+    }
+
+    public function generate()
+    {
+
+        //return "title".$this->Filter->generate();

```

Comment:
`generate` really shouldn't be empty, also you haven't addressed the comments I made before about where values are initialized.